src/utilities.ts

```diff
@@ -157,3 +158,29 @@ export function getRequestInformation(
         }
     }
 }
+
+export async function encryptKMS(plaintext: string, keyId: string): Promise<string> {
+    const kms = new AWS.KMS();
+    const params = {
+        KeyId: keyId, // `${process.env.LOGGING_MIDDLEWARE_KMS_KEY}`
+        Plaintext: plaintext,
+    };
+    const encryptRes = await kms.encrypt(params).promise();
+    if (Buffer.isBuffer(encryptRes.CiphertextBlob)) {
+        return Buffer.from(encryptRes.CiphertextBlob ? encryptRes.CiphertextBlob : 'undefined').toString('base64');
```

Comment:
Can we do `Buffer.from(encryptRes.Ciphert ...` directly without the if check? If it's not a buffer, then it should throw error automatically.